manifest.json

```diff
@@ -1,16 +1,25 @@
 {
   "name": "Jitsi Meet",
   "id": "jitsi",
+    "packaging_format": 1,
```

Comment:
You have an indentation problem on all this file

Reply:
indentation fixed